chainbridge/Cargo.toml

```diff
@@ -12,24 +12,24 @@ codec = { package = "parity-scale-codec", version = "3.0.0", default-features =
 serde = { version = "1.0.102", optional = true }
 
 # primitives
-sp-io = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, tag = "polkadot-v1.1.0" }
-sp-core = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, tag = "polkadot-v1.1.0" }
-sp-runtime = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, tag = "polkadot-v1.1.0" }
-sp-std = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, tag = "polkadot-v1.1.0" }
+sp-io = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, branch = "release-polkadot-v1.1.0" }
```

Comment:
Using "tag" will point to a fixed commit different from the last one used later by us in a centrifuge.